modules/installation-aws-add-iam-roles.adoc

```diff
@@ -39,3 +39,8 @@ controlPlane:
       iamRole: ExampleRole
 ----
 . Save the file and reference it when installing the {product-title} cluster.
+
+[NOTE]
+====
+To change or update an IAM account after the cluster has been installed, see the KCS article link:https://access.redhat.com/solutions/4284011[RHOCP 4 AWS cloud-credentials access key is expired].
```

Comment:
```suggestion
To change or update an IAM account after the cluster has been installed, see link:https://access.redhat.com/solutions/4284011[RHOCP 4 AWS cloud-credentials access key is expired] (Red Hat Knowledge Base).
```
Per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links).
And guidance in the contributor's guide around link formatting. I'll find that.

Reply:
Here it is https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute

Reply:
Thanks, I never really link KCS articles so I was not familiar with the conventions/rules.